database-plugins/src/main/java/io/cdap/plugin/db/batch/source/DBSource.java

```diff
@@ -352,6 +352,14 @@ private String getBoundingQuery() {
 
     @SuppressWarnings("checkstyle:WhitespaceAround")
     private void validate(FailureCollector collector) {
+      if (Boolean.TRUE.equals(getUseConnection())) {
```

Comment:
more natural if the getter returns a boolean, then you don't need to use equals()

Reply:
done :)